examples/ceed/ex1-volume.h

```diff
@@ -14,6 +14,10 @@
 // software, applications, hardware, advanced system engineering and early
 // testbed platforms, in support of the nation's exascale computing imperative.
 
+#ifndef ex1_volume_h
+#define ex1_volume_h
+#include <ceed.h>
```

Comment:
Do we want to include `ceed.h` in all of these? Is this to silence some IDEs?

Reply:
While we're at it, do we want to guard the gallery QFs? Just in case I guess?

Reply:
Yes, it is annoying in my IDE, because it gives all these warnings about unknown types, since there is not include directive in the file directly
![Screenshot from 2020-04-10 20-35-59](https://user-images.githubusercontent.com/39932030/79033402-3ccce700-7b6b-11ea-8862-a03d6378ac8b.png)

Reply:
I will find a way to change this setting

Reply:
> While we're at it, do we want to guard the gallery QFs? Just in case I guess?

I added them. Just to be extra safe in case any of us accidentally includes any of those files more than once. Honestly, since those are not files that the users must use/include, I don't think the risk was high.

Reply:
> I will find a way to change this setting

I couldn't find a way to make my IDE happy about the missing include in the same file. Perhaps @v-dobrev knows, since he mentioned he uses the same IDE?

Reply:
Maybe this will work: try adding the flags `-include ceed.h` or `-include /path/to/ceed.h` to the project `*.cflags` file. This, however, will probably apply to all c files in the project.

Reply:
Thanks @v-dobrev. Initially I tried to add it as a new line in the `*.cflags` file. Just for future reference: it doesn't work if you put the `-include` in a new line. But it works only if all flags are all the same line. For instance, I have
```
-std=c17 -include ceed.h
```
I was able to figure this out by looking at the [IDE bug report
](https://bugreports.qt.io/browse/QTCREATORBUG-19668)